src/aspire/commands/orient3d.py

```diff
@@ -84,4 +84,4 @@ def orient3d(
     logger.info("Save Estimate rotation matrices.")
     orient_est_src = orient_est.save_rotations()
 
-    orient_est_src.save_metadata(starfile_out, new_mrcs=False)
```

Comment:
Just so I understand, the old code was (optionally) not saving the images backing a starfile, only saving the updated starfile?

Reply:
It's not really clear to me what exactly this flag was supposed to do. Same with `overwrite`. This is as good an opportunity as any to try to clean this up

Reply:
okay, so the only place it was false was `orient_est_src.save_metadata(starfile_out, new_mrcs=False)` which you removed.  I wonder why that was used there.  If you don't think we need it I'm okay to remove it. Thanks!

IIRC Overwrite was to allow overwriting the (mrc) files if they existed (or not) which could be a very long/undesirable thing to do... Maybe these flags were related.

Reply:
Do we want flags separately for overwriting the STAR file and/or the MRCS? It's a little tricky to me because the user supplies the starfile filename, which is then used as the stem for the MRCS filenames anyway. By default it seems that unless you gave the exact same filename (unlikely since data sets have long cumbersome filenames typically), you would be creating new files

Reply:
Let's talk about this at our standup.  Mainly I'd like to figure out exactly what the old code did or intended to do (😅 ) first.  Then maybe what makes sense.

Naively, I'm thinking if we have separate save mrc and save meta (starfile) methods for an advanced user, they could just be called in a general save method that would be the most common use case. Maybe we do already and I just don't understand.  I haven't worked with this area of code in two years or so 😇